### ISSUE TITLE
mint issue

### ISSUE THREAD

**Issue by slooonshop** [opensolmap/solmap#21]
hi! when trying to mint solmap I see this error: 
Error: RPC response error -32002: Transaction simulation failed: Error processing Instruction 1: custom program error: 0x7dc [17 log messages]
Caused by: RPC response error -32002: Transaction simulation failed: Error processing Instruction 1: custom program error: 0x7dc [17 log messages]

Is that because you guys haven't started the next season yet or am I doing something wrong? RPC is working as well as I have balance on my wallet. I could also see that you have mint supply for the first season set in the code and I'm trying to mint below that number. I did use check-minted before that and it says 'not minted' so that's also not the case.

**Comment by samuelvanderwaal** [comment]
Yep, Season 1 is minted out and so minting is closed and it's going to be up to the community regarding a Season 2. I'm in the process of updating the contract so it can support a community DAO so the community can start to make decisions about the project's future.
